warehouse/manage/forms.py

```diff
@@ -36,7 +40,17 @@ class RoleNameMixin:
 
     role_name = wtforms.SelectField(
         "Select role",
-        choices=[("", "Select role"), ("Maintainer", "Maintainer"), ("Owner", "Owner")],
+        choices=[("", "Select role"), ("Maintainer", "Upload"), ("Owner", "Admin")],
```

Comment:
We should leave the labels for existing non-org projects as `Maintainer` and `Owner` until further notice, as merging this would change these labels and cause confusion for users.

Reply:
Fixed with commit [26bbc2e](https://github.com/pypi/warehouse/pull/11665/commits/26bbc2e824ce689cb4281287d0bec3e9dfb0b3fc).